tool/tsh/tsh.go

```diff
@@ -1389,11 +1389,12 @@ func onLogin(cf *CLIConf) error {
 				return trace.Wrap(err)
 			}
 			if err := updateKubeConfig(cf, tc, ""); err != nil {
```

Comment:
This empty if branch is really confusing to me. Let's invert the check and get rid of the else.

```
if err := updateKubeConfig(cf, tc, ""); err == nil {
    return trace.Wrap(onStatus(cf))
}
```